test/commons/table/is-data-cell.js

```diff
@@ -40,4 +40,42 @@ describe('table.isDataCell', function () {
 		assert.isFalse(axe.commons.table.isDataCell(target));
 	});
 
+	it('should ignore TDs with a valid role other than (grid)cell', function () {
+		fixture.innerHTML = '<table>' +
+			'<tr><td id="target1" role="columnheader">heading</td></tr>' +
+			'<tr><td id="target2" role="rowheader">heading</td></tr>' +
+			'<tr><td id="target3" role="presentation">heading</td></tr>' +
+			'</table>';
+
+		var target1 = $id('target1');
+		var target2 = $id('target2');
+		var target3 = $id('target3');
+		assert.isFalse(axe.commons.table.isDataCell(target1));
+		assert.isFalse(axe.commons.table.isDataCell(target2));
+		assert.isFalse(axe.commons.table.isDataCell(target3));
+	});
+
+	it('should return true for elements with role="(grid)cell"', function () {
```

Comment:
`role="(grid)cell"` - what is the intention behind the brackets around (grid)?

Reply:
`role=gridcell` and `role=cell` I don't see the point of separate cases. It's the same functionality. It's just data.